data/api/application-service/definitions/security.yaml

```diff
@@ -13,6 +14,6 @@
 # limitations under the License.
 homeserverAccessToken:
   type: apiKey
-  description: The `hs_token` provided by the application service's registration.
-  name: access_token
-  in: query
+  name: Authorization
+  in: header
+  description: The `Bearer` `hs_token` provided by the application service's registration.
```

Comment:
sigh. openapi 3.0 has better support for bearer tokens (https://swagger.io/docs/specification/authentication/bearer-authentication/). It really feels like we need to sort out https://github.com/matrix-org/matrix-spec/issues/331.

Reply:
indeed :( 

https://github.com/matrix-org/matrix-spec/pull/1200/commits/c510b1254022646803fb29fde96474d873cf8e89 was me thinking I could use v3 but reverting to v2